lib/Page.js

```diff
@@ -267,9 +268,17 @@ Page.prototype.insertSiteNav = function (pageData) {
   const siteNavMappedData = nunjucks.renderString(siteNavContent, userDefinedVariables);
   // Convert to HTML
   const siteNavDataSelector = cheerio.load(siteNavMappedData);
-  const siteNavHtml = md.render(siteNavDataSelector('markdown').html().trim());
+  if (siteNavDataSelector('navigation').length !== 1) {
+    if (siteNavDataSelector('navigation').length === 0) {
+      throw new Error(`<navigation> tag not found in ${siteNavPath}`);
```

Comment:
While we can certainty enforce users against multiple `<navgiation>` tags, we should consider not enforcing the user to use the tag compulsory. Sometimes the users might want to experiment around with things, or he may simply not want to include a `<navigation>` that can be rendered by MarkBind.